docs/api/web-contents.md

```diff
@@ -1416,6 +1420,46 @@ Executes the editing command `selectAll` in web page.
 
 Executes the editing command `unselect` in web page.
 
+#### `contents.scrollToTop()`
+
+Scrolls to the top of the current `webContents`.
+
+#### `contents.scrollToBottom()`
+
+Scrolls to the bottom of the current `webContents`.
+
+#### `contents.adjustSelection(options)`
+
+* `options` Object
+  * `start` Number - Index at which to begin the adjustment.
+  * `end` Number - Index at which to end the adjustment.
+
+Adjusts the current text selection starting and ending points in the focused frame by the given amounts. A negative amount moves the selection towards the beginning of the document, and a positive amount moves the selection towards the end of the document.
```

Comment:
nit: This might be too trivial, but it could be helpful to document the behaviour if the numbers end up giving a "negative" selection. Do we throw an error, or is it clamped to zero?

Reply:
@erickzhao i'm a little confused by what you mean 🤔  I noted that if the selection is negative that the selection moves towards the beginning of the document - is there a specific scenario that you're thinking of beyond that?

Reply:
I guess if you have:

> The quick brown fox _**jumps over**_ the lazy dog

And you pass in something wacky like `win.webContents.adjustSelection({ start: 10, end: -10 })`

Do you just have no selection altogether?

Reply:
@erickzhao correct - there will be no selection at all